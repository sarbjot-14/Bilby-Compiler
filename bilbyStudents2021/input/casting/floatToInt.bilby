main{
	imm nickel := 11.5;
	print [nickel as int] $n;
	
	
	
}
