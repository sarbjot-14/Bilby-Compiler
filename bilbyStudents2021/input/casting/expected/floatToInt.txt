Bad operand type for Printf instruction at location 17.
